README.md

```diff
@@ -3,12 +3,50 @@ Cyface Android SDK
 
 This project contains the Cyface Android SDK which is used by Cyface applications to capture data on Android devices.
 
-- [How to integrate the SDK](#how-to-integrate-the-sdk)
-- [Migration from Earlier Versions](#migration-from-earlier-versions)
+- [Integration Guide](#integration-guide)
+- [API Usage Guide](#api-usage-guide)
+- [Migration Guide](#migration-guide)
+- [Developer Guide](#developer-guide)
 - [License](#license)
 
 
-How to integrate the SDK
+Integration Guide
+---------------------
+
+This library is published to the Github Package Registry.
+
+To use it's submodules as a dependencies you need to:
+
+1. Make sure you linked the repository:
+
+    * Add the custom repository in your `build.gradle`:
+
+    ``` 
+    repositories {
+        // Other maven repositories, e.g.:
+        jcenter()
+        google()
+        // Repository for this library
+        maven {
+            url = uri("https://maven.pkg.github.com/cyface-de/android-backend")
+        }
+    }
+    ```
+    
+2. Add this package as a dependency to your app's `build.gradle`:
+
+    ```
+    dependencies {
+        implementation "de.cyface:datacapturingCyfaceFull:$cyfaceBackendVersion"
+        implementation "de.cyface:synchronizationCyfaceFull:$cyfaceBackendVersion"
+        implementation "de.cyface:persistenceCyfaceFull:$cyfaceBackendVersion"
```

Comment:
I changed the artifactId naming in a later commit to:
- "submoduleName" or "submoduleNameMovebis"
- i.e. only the "Full" "Release" variants for "cyface" and "movebis" are published. If we notice that we need to publish all variants like mock and debug I'll change this when needed.
- I wanted to keep the artifact names simple, this is why I don't name e.g. "datacapturingCyface" but "datacapturing" as the goal is probably to only have this eventually (i.e. to avoid datacapturingMovebis, etc.)